frontend/src/component/common/DialogFormTemplate/DialogFormTemplate.styles.tsx

```diff
@@ -16,16 +15,19 @@ export const StyledFormSection = styled('div')(({ theme }) => ({
 
 export const TopGrid = styled(StyledFormSection)(({ theme }) => ({
     display: 'grid',
-    gridTemplateAreas:
-        '"icon header" "icon project-name" "icon project-description"',
+    gridTemplateAreas: `
+        "icon header"
+        ".    project-name"
+        ".    project-description"`,
     gridTemplateColumns: 'auto 1fr',
     gap: theme.spacing(4),
 }));
 
-export const StyledIcon = styled(ProjectIcon)(({ theme }) => ({
-    fill: theme.palette.primary.main,
-    stroke: theme.palette.primary.main,
-}));
+export const styleIcon = (Icon: React.ComponentType) =>
```

Comment:
Why such lower case naming and code style change?

Reply:
Not sure what the code style change you're referring to is, but I lowercased it because it's a function now, and not a component directly. Happy to uppercase it if you think that's better, though 🤷🏼

Reply:
Oh, and it's a function because we don't know what kind of icon to use here, so we need to create this dynamically. This allows us to do that formatting in the template component instead of on the outside.

Reply:
ah yes its a function now, I think it is fine then